From ca511024f394a56b86a861e6029585ec1448f789 Mon Sep 17 00:00:00 2001
From: Markus Raab <elektra@markus-raab.org>
Date: Thu, 2 Jul 2015 09:45:15 +0200
Subject: [PATCH 7/7] fix #230

allow to use qt-gui with qt built with -reduce-relocations
---
 src/tools/qt-gui/CMakeLists.txt | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/src/tools/qt-gui/CMakeLists.txt b/src/tools/qt-gui/CMakeLists.txt
index 270912a..4915c3c 100644
--- a/src/tools/qt-gui/CMakeLists.txt
+++ b/src/tools/qt-gui/CMakeLists.txt
@@ -12,6 +12,11 @@ elseif (NOT DISCOUNT_FOUND)
 	remove_tool(qt-gui "discount (libmarkdown) not found")
 else()
 
+if (Qt5_POSITION_INDEPENDENT_CODE)
+	# Qt was built with -reduce-relocations
+	set(CMAKE_POSITION_INDEPENDENT_CODE ON)
+endif()
+
 include_directories (
 	${CMAKE_SOURCE_DIR}
 	${CMAKE_CURRENT_BINARY_DIR}
-- 
2.3.6

